OWNERS.md

```diff
@@ -39,14 +39,15 @@ Being part of Technical Steering Committee (TSC) [@StackStorm/maintainers](https
   - Systems, ST2 Exchange. [Case Study](https://stackstorm.com/case-study-bitovi/).
 * Nick Maludy ([@nmaludy](https://github.com/nmaludy)) <<nmaludy@gmail.com>>
   - Community, Core, Systems, StackStorm Exchange, Puppet deployment. [Case Study](https://stackstorm.com/case-study-dmm/).
+* Carlos ([@nzlosh](https://github.com/nzlosh))
+  - Chatops, Errbot, Community, Discussions, StackStorm Exchange
```

Comment:
As I understand at least st2 core needs to be added to this list.

Any other areas of interest? WDYT @nzlosh ?

Reply:
Focusing on the st2 core along with the existing subjects should be enough for now.

Reply:
Awesome! 
As discussed internally, we'd like to see first more pro-active effort and contribution activity to the st2 core platform. Can you work a bit to improve the https://github.com/StackStorm/st2/commits?author=nzlosh?